components/dashboard/src/teams/TeamSettings.tsx

```diff
@@ -82,7 +122,39 @@ export default function TeamSettings() {
                 title="Settings"
                 subtitle="Manage general team settings."
             >
-                <h3>Delete Team</h3>
+                <h3>Team Name</h3>
+                <p className="text-base text-gray-500 max-w-2xl">
+                    This is your team's visible name within Gitpod. For example, the name of your company.
+                </p>
+                {errorMessage && (
+                    <Alert type="error" closable={true} className="mb-2 max-w-xl rounded-md">
+                        {errorMessage}
+                    </Alert>
+                )}
+                {updated && (
+                    <Alert type="message" closable={true} className="mb-2 max-w-xl rounded-md">
+                        Team name has been updated.
+                    </Alert>
+                )}
+                <div className="flex flex-col lg:flex-row">
+                    <div>
+                        <div className="mt-4 mb-3">
+                            <h4>Name</h4>
+                            <input type="text" value={teamName} onChange={onNameChange} />
+                        </div>
+                    </div>
+                </div>
+                <div className="flex flex-row">
+                    <button
+                        className="primary"
+                        disabled={team?.name === teamName || !!errorMessage}
```

Comment:
nit: currently, the user is able to change the team name to the same name just with whitespaces around it.

```suggestion
                        disabled={team?.name === teamName.trim() || !!errorMessage}
```